freud/environment.pyx

```diff
@@ -373,17 +369,17 @@ cdef class LocalDescriptors(PairCompute):
             nlist.get_ptr(), dereference(qargs.thisptr))
         return self
 
-    @Compute._computed_property()
+    @Compute._computed_property
     def nlist(self):
-        return freud.locality.nlist_from_cnlist(self.thisptr.getNList())
+        return freud.locality._nlist_from_cnlist(self.thisptr.getNList())
```

Comment:
Should this be a static method / class method of `NeighborList`? Similarly for `BoxFromCPP` and `freud.box.Box`.

Reply:
Yes, I think that makes sense. However, since neither of these methods is part of the public API I just made an issue, we can wait on this for now.